<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Alina Selega</title>
    <link rel="stylesheet" href="styles.css"> 
</head>
<body>
    <header>
        <div class="container">
            <h1>Alina Selega</h1>
        </div>
    </header>

    <div class="banner">
    <div class="container">
            <a href="index.html">About Me</a>
            <a href="research.html">Research</a>
            <a href="publications.html">Publications</a>
    </div>
    </div>

    <section id="profile">
        <div class="container">
            <img src="picture.jpg" alt="Alina Selega" class="profile-pic">
        </div>
        <div class="container">
            <a href="https://scholar.google.ca/citations?user=tpG4CaIAAAAJ&hl=en&oi=ao">Google Scholar</a>
            <a href="https://github.com/alinaselega">GitHub</a>
            <a href="https://www.linkedin.com/in/alina-selega-753860235/">LinkedIn</a>
            <a href="https://openreview.net/profile?id=~Alina_Selega1">OpenReview</a>
        </div>

    </section>

    <section id="publications">
        <div class="container">
            <h2>Publications</h2>

            <ul>
                Lee Y, Chen E, Chan D, Dinesh A, Afiuni-Zadeh S, Klamann C, <b>Selega A</b>, Mrkonjic M, Jackson H, Campbell K. "Segmentation error aware clustering for highly multiplexed imaging."
                <i>bioRxiv</i> (2024). <a href="https://www.biorxiv.org/content/10.1101/2024.02.29.582827v1">DOI</a>
            </ul>

            <ul>
                Fang C, <b>Selega A*</b>, Campbell K*. "Beyond benchmarking and towards predictive models of dataset-specific single-cell RNA-seq pipeline performance."
                <i>Genome Biology</i> 25 (2024): 159. <a href="https://doi.org/10.1186/s13059-024-03304-9">DOI</a> *jointly supervised
            </ul>

            <ul>
                <b>Selega A</b> and Campbell K. "Multi-objective Bayesian optimization with heuristic objectives for biomedical and molecular data analysis workflows."
                <i>Transactions on Machine Learning Research</i> (2023): 2835-8856. <a href="https://openreview.net/forum?id=QspAcsAyis">URL</a>
            </ul>

            <ul>
                Harrigan C, Rubanova Y, Morris Q, <b>Selega A.</b> "TrackSigFreq: subclonal reconstructions based on mutation signatures and allele frequencies."
                <i>Pacific Symposium on Biocomputing</i> 25 (2020): 238-249. <a href="https://doi.org/10.1142/9789811215636_0022">DOI</a>
            </ul>

            <ul>
                <b>Selega A</b>, Sirocchi C, Iosub I, Granneman S, Sanguinetti G. "Robust statistical modeling improves sensitivity of high-throughput RNA structure probing experiments."
                <i>Nature Methods</i> 14 (2017): 83-89. <a href="https://doi.org/10.1038/nmeth.4068">DOI</a>
            </ul>

            <ul>
                van Nues R, Schweikert G*, de Leau E*, <b>Selega A*</b>, Langford A, Franklin R, Iosub I, Wadsworth P, Sanguinetti G, Granneman S. 
                "Kinetic CRAC uncovers a role for Nab3 in determining gene expression profiles during stress." <i>Nature Communications</i> 8.12 (2017). 
                <a href="https://doi.org/10.1038/s41467-017-00025-5">DOI</a> *contributed equally
            </ul>

            <ul>
                <b>Selega A</b> and Sanguinetti G. "Trends and challenges in Computational RNA biology." 
                <i>Genome Biology</i> 17 (2016): 253. <a href="https://doi.org/10.1186/s13059-016-1117-7">DOI</a> (Invited review)
            </ul>

            <h3>Thesis</h3>
            <ul>
            <b>Selega A.</b> "Computational methods for RNA integrative biology." <i>University of Edinburgh</i> (2018) <a href="https://era.ed.ac.uk/handle/1842/29630">URL</a>
            </ul>
        </div>
    </section>

</body>
</html>
